target/linux/ramips/dts/mt7621_zbtlink_zbt-we1326.dts

```diff
@@ -127,6 +127,11 @@
 		reg = <0x0000 0 0 0 0>;
 		mediatek,mtd-eeprom = <&factory 0x8000>;
 		ieee80211-freq-limit = <5000000 6000000>;
+		
```

Comment:
Small nit-pick: please clean whitespaces in empty lines, in both patch hunks, before merging